pkg/flow/grpc-instance-variables.go

```diff
@@ -5,92 +5,19 @@ import (
 	"context"
 	"errors"
 	"io"
-	"os"
 	"time"
 
 	"github.com/direktiv/direktiv/pkg/flow/bytedata"
 	"github.com/direktiv/direktiv/pkg/flow/database"
-	"github.com/direktiv/direktiv/pkg/flow/database/recipient"
-	"github.com/direktiv/direktiv/pkg/flow/ent"
-	entinst "github.com/direktiv/direktiv/pkg/flow/ent/instance"
-	entvardata "github.com/direktiv/direktiv/pkg/flow/ent/vardata"
-	entvar "github.com/direktiv/direktiv/pkg/flow/ent/varref"
-	derrors "github.com/direktiv/direktiv/pkg/flow/errors"
 	"github.com/direktiv/direktiv/pkg/flow/grpc"
 	"github.com/google/uuid"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"google.golang.org/protobuf/types/known/emptypb"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-func (srv *server) getInstanceVariable(ctx context.Context, cached *database.CacheData, key string, load bool) (*database.VarRef, *database.VarData, error) {
-	vref, err := srv.database.InstanceVariable(ctx, cached.Instance.ID, key)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	vdata, err := srv.database.VariableData(ctx, vref.VarData, load)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	return vref, vdata, nil
-}
-
-func (srv *server) getThreadVariable(ctx context.Context, cached *database.CacheData, key string, load bool) (*database.VarRef, *database.VarData, error) {
-	vref, err := srv.database.ThreadVariable(ctx, cached.Instance.ID, key)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	vdata, err := srv.database.VariableData(ctx, vref.VarData, load)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	return vref, vdata, nil
-}
-
 func (srv *server) traverseToInstanceVariable(ctx context.Context, namespace, instance, key string, load bool) (*database.CacheData, *database.VarRef, *database.VarData, error) {
-	id, err := uuid.Parse(instance)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-
-	cached := new(database.CacheData)
-
-	err = srv.database.Instance(ctx, cached, id)
-	if err != nil {
-		srv.logger.Errorf(ctx, srv.ID, srv.flow.GetAttributes(), "Failed to resolve instance %s", instance)
-		return nil, nil, nil, err
-	}
-
-	fStore, _, _, rollback, err := srv.flow.beginSqlTx(ctx)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-	defer rollback()
-
-	file, revision, err := fStore.GetRevision(ctx, cached.Instance.Revision)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-
-	cached.File = file
-	cached.Revision = revision
-
-	if cached.Namespace.Name != namespace {
-		return nil, nil, nil, os.ErrNotExist
-	}
-
-	vref, vdata, err := srv.getInstanceVariable(ctx, cached, key, load)
-	if err != nil {
-		srv.logger.Errorf(ctx, srv.ID, srv.flow.GetAttributes(), "Failed to resolve variable")
-		return nil, nil, nil, err
-	}
-
-	return cached, vref, vdata, nil
+	return nil, nil, nil, nil
```

Comment:
This function is still being called elsewhere, so it is missing a TODO.

Reply:
Fixed :)